<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>测评操作项</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>471</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>个人信息填写</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>451</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>学号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_Name">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>张三</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_StudentNum">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>2016222976</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>年级</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_Greates">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>2016</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>其他</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_Other">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>测评人姓名</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>471</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>基础素质评分统计</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_DeScores">
         <property name="text">
          <string>60</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>德育(小组评分)</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_TiScores">
         <property name="text">
          <string>99</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>体育</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>智育</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_ZhiScores">
         <property name="text">
          <string>99</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>德育（学办评分）</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_DeScores2">
         <property name="text">
          <string>40</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>471</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>发展素质评分计算</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>451</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>综 合 技 能</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pbt_AddOtherScores">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>创新创造能力</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>特 殊 表 彰</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pbt_AddScores">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>组织管理能力</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pbt_SMAdd">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>直接输入发展素质评分</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>注意</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>509</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pbt_ReSet">
        <property name="text">
         <string>重置</string>
        </property>
        <property name="icon">
         <iconset resource="iamges.qrc">
          <normaloff>:/images/Delete.ico</normaloff>:/images/Delete.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbt_Finish">
        <property name="text">
         <string>完成</string>
        </property>
        <property name="icon">
         <iconset resource="iamges.qrc">
          <normaloff>:/images/704.bmp</normaloff>:/images/704.bmp</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>451</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>评分情况预览</string>
    </property>
    <widget class="QTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>481</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>生成预览信息</string>
        </property>
        <property name="icon">
         <iconset resource="iamges.qrc">
          <normaloff>:/images/100.bmp</normaloff>:/images/100.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>清空预览</string>
        </property>
        <property name="icon">
         <iconset resource="iamges.qrc">
          <normaloff>:/images/706.bmp</normaloff>:/images/706.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>保存为文件</string>
        </property>
        <property name="icon">
         <iconset resource="iamges.qrc">
          <normaloff>:/images/104.bmp</normaloff>:/images/104.bmp</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_Name</tabstop>
  <tabstop>lineEdit_StudentNum</tabstop>
  <tabstop>lineEdit_Greates</tabstop>
  <tabstop>lineEdit_Other</tabstop>
  <tabstop>lineEdit_DeScores</tabstop>
  <tabstop>lineEdit_DeScores2</tabstop>
  <tabstop>lineEdit_ZhiScores</tabstop>
  <tabstop>lineEdit_TiScores</tabstop>
  <tabstop>pbt_AddScores</tabstop>
  <tabstop>pbt_AddOtherScores</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pbt_SMAdd</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pbt_ReSet</tabstop>
  <tabstop>pbt_Finish</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="iamges.qrc"/>
 </resources>
 <connections/>
</ui>
